packages/core/util/jexl.ts

```diff
@@ -97,13 +102,6 @@ export default function (/* config?: any*/): JexlNonBuildable {
   j.addFunction('trimEnd', (str: string) => str.trimEnd())
   j.addFunction('trimStart', (str: string) => str.trimStart())
 
-  j.addFunction('hashcode', (str: string) =>
-    str
-      .split('')
-      .map(c => c.charCodeAt(0))
-      .reduce((a, b) => a + b, 0),
-  )
-
```

Comment:
Just making sure, is Viewer.js in the jbrowse-protein-widget being removed? Cause thats the only place hashcode is used right now

Reply:
yeah it's gone in main now